osu.Game/Graphics/Containers/LogoTrackingContainer.cs

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.MathUtils;
+using osu.Game.Screens.Menu;
+using osuTK;
+
+namespace osu.Game.Graphics.Containers
+{
+    /// <summary>
+    /// A container that handles tracking of an <see cref="OsuLogo"/> through different layout scenarios.
+    /// </summary>
+    public class LogoTrackingContainer : Container
+    {
+        public Facade LogoFacade { get; }
+
+        /// <summary>
+        /// Whether or not the logo assigned to this FacadeContainer should be tracking the position of its facade.
+        /// </summary>
+        public bool Tracking
```

Comment:
I'm wondering whether we need this. How about:

```csharp
StartTracking(logo, facadeScale(?), duration, easing);
StopTracking();
```

As it is, I believe the following is currently happening

```csharp
c.Tracking = true;
c.SetLogo(logo);

Schedule(() =>
{
    c.Tracking = false;

    Scheduler.AddDelayed(() =>
    {
        c.Tracking = true;
        // In the next frame, logo instantly moves to the facade position
    }, 500);
});
```

Reply:
Could do. The method is entirely just sets anyways, even if it were to be used in every instance in ButtonSystem there's not much to be optimized there.